src/bin/initdb/initdb.c

```diff
@@ -2365,7 +2365,7 @@ usage(const char *progname)
 	printf(_("      --wal-segsize=SIZE    size of WAL segments, in megabytes\n"));
 	printf(_("\nLess commonly used options:\n"));
 	printf(_("  -d, --debug               generate lots of debugging output\n"));
-	printf(_("  -e  --enc-cipher=MODE     set encryption cipher for data encryption\n"));
+	printf(_("  -e  --enc-cipher=TYPE     set encryption cipher (AES-128/AES-256) for data encryption\n"));
```

Comment:
Listing available values would help user but I'm concerned about maintainability. If we do that we have to keep the list up to date. Since initdb command is executed only once at database cluster initialization I'm not sure it's worth to maintain it.
It's the same for the error message you changed.